app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, file_in, file_out = command.split()
```

Comment:
Make sure you check that the `cp` input command is correct.  It might be 'int' or start with, let's say, 'foo' instead of 'cp')